packages/core/src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -1,5 +1,39 @@
 {
     "types": [
+        {
+            "name": "cfnParameterFileUsed",
+            "type": "boolean",
+            "description": "Boolean value of whether or not a Cfn parameter file is provided."
```

Comment:
can you try and put all metrics in common only. makes it easier to keep track

Reply:
I created these to have the metrics available in my dev environment. I am still waiting for the common [changes](https://github.com/aws/aws-toolkit-common/pull/735) to be merged, which has the same metrics.